test/policy/relating_to_actor_via_test.exs

```diff
@@ -0,0 +1,438 @@
+        authorize_if(relating_to_actor_via(:does_not_exist))
+      end
+    end
+  end
+
+  defmodule BadPolicyRelPathName do
+    use Ash.Resource,
+      domain: Domain,
+      data_layer: Ash.DataLayer.Ets,
+      authorizers: [Ash.Policy.Authorizer]
+
+    actions do
+      default_accept(:*)
+      defaults([:read, :destroy, create: :*, update: :*])
+    end
+
+    ets do
+      private?(true)
+    end
+
+    attributes do
+      uuid_primary_key(:id)
+    end
+
+    policies do
+      policy always() do
+        authorize_if(relating_to_actor_via([:does_not_exist, :neither_does_this]))
+      end
+    end
+  end
+
+  defmodule BadPolicyTargetField do
+    use Ash.Resource,
+      domain: Domain,
+      data_layer: Ash.DataLayer.Ets,
+      authorizers: [Ash.Policy.Authorizer]
+
+    actions do
+      default_accept(:*)
+      defaults([:read, :destroy, create: :*, update: :*])
+    end
+
+    ets do
+      private?(true)
+    end
+
+    attributes do
+      uuid_primary_key(:id)
+    end
+
+    relationships do
+      belongs_to(:comment, Ash.Test.Policy.RelatingToActorViaTest.Comment) do
+        public?(true)
+      end
+    end
+
+    policies do
+      policy always() do
+        authorize_if(
+          relating_to_actor_via([:comment, :post, :user], target_field: :does_not_exist)
+        )
+      end
+    end
+  end
+
+  defp user_fixture do
+    User
+    |> Ash.Changeset.for_create(:create, %{c: "key1", d: "key2"})
+    |> Ash.create!()
+  end
+
+  setup do
+    user = user_fixture()
+
+    post =
+      Post
+      |> Ash.Changeset.for_create(:create)
+      |> Ash.Changeset.manage_relationship(:user, user, type: :append)
+      |> Ash.create!(authorize?: false)
+
+    maintainer =
+      Maintainer
+      |> Ash.Changeset.for_create(:create, %{a: "key1", b: "key2"})
+      |> Ash.Changeset.manage_relationship(:post, post, type: :append)
+      |> Ash.create!(authorize?: false)
+
+    comment =
+      Comment
+      |> Ash.Changeset.for_create(:create)
+      |> Ash.Changeset.manage_relationship(:post, post, type: :append)
+      |> Ash.create!(authorize?: false)
+
+    %{comment: comment, maintainer: maintainer, post: post, user: user}
+  end
+
+  describe "no options" do
+    test "relating_to_actor_via with create attribute", %{comment: comment, user: user} do
+      assert {:ok, _} =
+               Reaction
+               |> Ash.Changeset.for_create(:create, %{comment_id: comment.id})
+               |> Ash.create(actor: user)
+    end
+
+    test "relating_to_actor_via errors with other actor", %{comment: comment} do
+      user = user_fixture()
+
+      assert {:error, %Ash.Error.Forbidden{}} =
+               Reaction
+               |> Ash.Changeset.for_create(:create, %{comment_id: comment.id})
+               |> Ash.create(actor: user)
+    end
+
+    # TODO: how can we make this work?
+    @tag :skip
+    test "relating_to_actor_via with manage_relationship", %{comment: comment, user: user} do
```

Comment:
I don't think there is any good way to make this work past one level. At least not one that isn't super complicated. This is how we do it in the other `relating_to_actor` check:

```elixir
      case changeset.relationships[relationship.name] do
        [{[input], opts}] ->
          primary_key = Ash.Resource.Info.primary_key(relationship.destination)

          actor_keys = take_keys(actor, primary_key)
          input_keys = take_keys(input, primary_key)

          opts[:on_lookup] == :relate and Enum.all?(actor_keys, & &1) &&
            Enum.all?(input_keys, & &1) and input_keys == actor_keys

        _ ->
          false
      end
```

Actually...you could probably do this...it's still really complex though. 

You'd need to:
1. find the input that corresponds to either the attribute backing the relationship
2. *or* find any argument that has a `manage_relationship` change for the given relationship w/ a `on_lookup: :relate`.

You'd have to check this recursively. It's not pretty and also probably not worth our time for now. I'd suggest just adding a check that explicitly shows an error if the first relationship is being managed.